turbinia/workers/bulk_extractor.py

```diff
@@ -72,6 +72,24 @@ def run(self, evidence, result):
       return result
     return result
 
+  def check_xml_attrib(self, xml_key):
+    """Checks if a key exists within the xml report
+
+    Args: 
+      xml_key(str): the xml key to check for.
+    
+    Returns:
+      xml_hit(str): the xml value else return N/A.
+    """
+    xml_hit = "N/A"
```

Comment:
Another nit, but maybe change to single quotes to match (this should also get picked up by the linter).

Reply:
Fixed!